src/components/Competition/PrintingManager/Scorecards/Scorecards.js

```diff
@@ -63,6 +68,33 @@ const Scorecards = ({ wcif }) => {
   const isSelectionEmpty =
     selectedRounds.length === 0 || selectedRooms.length === 0;
 
+  const [language, setLanguage] = useState('en');
+
+  const LanguageSelector = ({ language, setLanguage, label }) => {
```

Comment:
We should move the component outside, but since we use it in a single place, I think it's fine to just inline it.